internal/tool/golang.go

```diff
@@ -0,0 +1,128 @@
+package tool
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/samber/lo"
+)
+
+func patchGoModFilesForStdlib(srcDir string, files []string) string {
+	// Copy the files to a temporary directory because /src is read-only
+	dstDir := "/tmp/src"
+	if err := CopyFiles(files, srcDir, dstDir); err != nil {
```

Comment:
This seems like it can become a bottleneck...
Since it's already being done file by file can't it be changed to instead create a symlink for every file other than go.mod? And for those, you actually copy and change the content.

Reply:
It seems a fair assumption.
Would be safer to prevent future changes though. Won't we leverage dockers scanning somewhere in the future?